tests/scripts/localPathPVForMon.sh

```diff
@@ -51,9 +41,9 @@ spec:
 apiVersion: v1
 kind: PersistentVolume
 metadata:
-  name: local-vol2
+  name: local-mon2
```

Comment:
The test will be nice with one mon so we don't need to duplicate all these PVs.

Reply:
I'lll fix it too.